docs/upgrading-fcos.md

```diff
@@ -13,14 +13,125 @@ Occasionally, changes are made to Fedora CoreOS Butane configs (those that speci
 1. TOC
 {:toc}
 
-{% comment %}
-
 ## From Version 1.4.0 to Version 1.5.0
 
 There are no breaking changes between versions 1.4.0 and 1.5.0 of the `fcos` configuration specification. Any valid 1.4.0 configuration can be updated to a 1.5.0 configuration by changing the version string in the config.
 
 The following is a list of notable new features.
 
+## GRUB password
+
+A new `grub` section was added with an array of users can be set with `name`(s) and `password_hash`(s). Any user that does not have the given usernames and passwords will not be able to use the GRUB command line, modify kernel command-line arguments, or boot non-default OSTree deployments.
```

Comment:
Maybe:

```suggestion
The config gained a new top-level `grub` section. It contains a `users` section with a list of usernames and corresponding password hashes for authenticating to the GRUB bootloader. If any users are specified, GRUB will require authentication before using the GRUB command line, modifying kernel command-line arguments, or booting non-default OSTree deployments. Password hashes can be generated with `grub2-mkpasswd-pbkdf2`.
```

We should also document the command elsewhere; PR in #445.

Reply:
Thats much better, thank you for the wording fixes.